docs/recipes/controlplane/tp-cp.yaml

```diff
@@ -516,7 +538,7 @@ helmCharts:
                     pathType: Prefix
                     backend:
                       service:
-                        name: hybrid-proxy
+                        name: hybrid-proxy-${CP_INSTANCE_ID}
```

Comment:
Hi @dgidwani-tibco  I see we add this CP_INSTANCE_ID but I don't see the nginx annotation for non https backend.

Reply:
is this for non local ( aws/azure ) provider ?